markdown/pxf/PXFExternalTableandAPIReference.html.md.erb

```diff
@@ -885,7 +923,7 @@ public class Constant
 
 #### <a id="filterobject"></a>Filter Object
 
-Filter Objects can be internal, such as those you define; or external, those that the remote system uses. For example, for HBase, you define the HBase `Filter` class (`org.apache.hadoop.hbase.filter.Filter`), while for Hive, you use an internal default representation created by the PXF framework, called `BasicFilter`. You can decide the filter object to use, including writing a new one. `BasicFilter` is the most common:
+Filter Objects can be internal, such as those you define; or external, those that the remote system uses. For example, for HBase, you define the HBase `Filter` class (`org.apache.hadoop.hbase.filter.Filter`), while for Hive, you use an internal default representation created by the PXF framework, called `BasicFilter`. You can choose the filter object to use, including writing a new one. `BasicFilter` is the most common:
```

Comment:
Change the ; to a , (or use dashes as instead like -such as those you define-)

Also remove the commas after HBase, and Hive, in the second sentence.